packages/components/app/styles/components/alert.scss

```diff
@@ -14,6 +14,30 @@
   code, pre {
     font-family: var(--token-typography-font-stack-code);
   }
+
+  // Manually copied from components/link/standalone.scss for -primary
```

Comment:
@didoo This can be a mixin (see https://github.com/hashicorp/design-system/commit/625831fe122045011a3054d4d76e3b365acef473 where I did some exploration), but there will need to be a few mixins because of `.hds-link-standalone__text`). Wanted to gut check if that approach is what you want? Can make that change if so

Reply:
no, the styling is different, this should be the styling for an "inline" link (see https://www.figma.com/file/noyY6dUMDYjmySpHcMjhkN/HDS-Product---Components-%5BWIP%5D?node-id=2365%3A21590)

I'll push directly the changes in this PR

Reply:
Changes done in https://github.com/hashicorp/design-system/pull/99/commits/b0e21dd6d12fbfd3954ecea8499638b8eb7961cc